src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -1371,6 +1376,20 @@ public function getContentType()
         return $this->getFormat($this->headers->get('CONTENT_TYPE'));
     }
 
+    /**
+     * Gets the acceptable client formats associated with the request.
+     *
+     * @return array List of acceptable formats by the client
+     */
+    public function getAcceptableFormats()
```

Comment:
missing return type hint (and removal of the corresponding docblock line since it will become useless)